### PR TITLE
[Add] abandoned_carts module.

### PR DESCRIPTION
**Pull request by nikunjantala**

### CODE REVIEW

**Comment by jans23** [comment]
@alihashaam Could you fix the CI?

**Comment by jans23** [comment]
@nikunjantala Is this ready for review? Please add such label.

File: abandoned_carts/wizard/customer.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    #                'name': record_name,
+    #                'date': current_date,
+    #                'res_model': 'res.partner',
+    #                'res_id': record_id,
+    #                'user_id': user_id,
+    #                'error': error
+    #            })
+    #            _LOGGER.info('name %s, date %s, model %s, res_id %s, user %s'
+    #            %(record_name, current_date, 'res.partner', record_id, user.name))
+    #            self._cr.execute('RELEASE SAVEPOINT remove_partner')
```
Comment (alihashaam) [inline abandoned_carts/wizard/customer.py]: remove comments please

**Comment by alihashaam** [comment]
@nikunjantala 

Can you please fix isort warnings and uncomment the comments.

or the changes are quite small and non-functional so I can also do those changes. whatever you prefer

**Comment by jans23** [comment]
@alihashaam Feel free to do it.

**Comment by alihashaam** [comment]
ok @jans23

**Comment by alihashaam** [comment]
@jans23 CI issues are fixed now and review is done

**Pull request merged**